inference-engine/src/transformations/include/transformations/common_optimizations/reshape_sequence_fusion.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API ReshapeSequenceFusion;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief ReshpaeSequenceFusion fuses sequence of Reshape operation into single Reshape
+ */
+
+class ngraph::pass::ReshapeSequenceFusion: public ngraph::pass::MatcherPass {
+public:
+    NGRAPH_RTTI_DECLARATION;
```

Comment:
Can we use new RTTI?

Reply:
Currently all transformations (100+) are using this type of declaration and ngraph namespaces.
I insist on making single shot transition in dedicated PR where we change RTTI, namespaces and other related things. And now I don't want to have a difference in existing approach not to confuse library developers and its customers.